src/main/java/manager/DatabaseLoader.java

```diff
@@ -63,155 +76,166 @@
     }
 
     private static String loadWeapons(Statement statement, Semaphore semaphore) {
-        try (ResultSet weaponSet = statement.executeQuery("SELECT * FROM Items NATURAL JOIN Waffen")) {
-            ObservableList<Weapon> weaponList = FXCollections.observableArrayList();
+        ObservableList<Weapon> weaponList = FXCollections.observableArrayList();
+
+        try (ResultSet weaponSet = statement.executeQuery(format("SELECT * FROM %s NATURAL JOIN %s", "table.items", "table.weapons"))) {
             while (weaponSet.next()) {
                 Weapon weapon = new Weapon();
-                weapon.setName(getString(weaponSet, "Bezeichnung"));
-                weapon.setMaterial(getString(weaponSet, "Material"));
-                weapon.setTyp(getString(weaponSet, "Typ"));
-                weapon.setSubTyp(getString(weaponSet, "Subtyp"));
-                weapon.setRequirement(getString(weaponSet, "Vorraussetzung"));
-                weapon.setInitiative(getString(weaponSet, "Initiative"));
-                weapon.setDice(getString(weaponSet, "Würfel/Belastung"));
-                weapon.setDamage(weaponSet.getInt("Schaden/Schutz"));
-                weapon.setHit(weaponSet.getInt("Treffer"));
-                weapon.setEffect(getString(weaponSet, "Effekt"));
-                weapon.setSlots(weaponSet.getInt("Verbesserungsslots"));
-                weapon.setRarity(getString(weaponSet, "Seltenheit"));
-                weapon.setCost(getString(weaponSet, "Preis"));
-                weapon.setTier(weaponSet.getInt("Tier"));
+                weapon.setName(getString(weaponSet, getLocalized("column.name")));
```

Comment:
is this a weapon specififc column.name? Or is this a generic column.name for every database table? This should be configurable for every table individually.

Reply:
A consistent naming scheme is easier to maintain.
In this way I force the user to use one.